tests/test_acl.py

```diff
@@ -207,6 +207,66 @@ def test_AclRuleL4SrcPort(self, dvs):
         (status, fvs) = atbl.get(acl_entry[0])
         assert status == False
 
+    def test_AclRuleInOutPorts(self, dvs):
+        """
+        hmset ACL_RULE|test|acl_test_rule priority 55 PACKET_ACTION FORWARD IN_PORTS Ethernet0,Ethernet4 OUT_PORTS Ethernet8,Ethernet12
+        """
+
+        db = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+
+        # create acl rule
+        tbl = swsscommon.Table(db, "ACL_RULE")
+        fvs = swsscommon.FieldValuePairs([("priority", "55"),
+                                          ("PACKET_ACTION", "FORWARD"),
+                                          ("IN_PORTS", "Ethernet0,Ethernet4"),
+                                          ("OUT_PORTS", "Ethernet8,Ethernet12")])
+        tbl.set("test|acl_test_rule", fvs)
+
+        time.sleep(1)
+
+        test_acl_table_id = self.get_acl_table_id(dvs, adb)
+
+        # check acl table in asic db
+        atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_ENTRY")
+        keys = atbl.getKeys()
+
+        acl_entry = [k for k in keys if k not in dvs.asicdb.default_acl_entries]
+        assert len(acl_entry) == 1
```

Comment:
It would be nice to have a comment here to state which entry is expected to be missing.